src/Microsoft.Health.Dicom.Client/Models/UpdateExtendedQueryTagEntry.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Client.Models
+{
+    /// <summary>
+    /// Encapsulate parameters for updating extended query tag.
+    /// </summary>
+    public class UpdateExtendedQueryTagEntry
+    {
+        /// <summary>
+        /// Gets or sets query status.
+        /// </summary>
+        public QueryStatus QueryStatus { get; set; }
```

Comment:
Do we want to be strict when parsing these values? For examples, should we throw an exception if a user thinks they can update other fields in the tag by including things like:
```json
{
  "TagPath": "foo",
  "QueryStatus": "Enabled"
}
```

I can see for `System.Text.Json` we can add a property with `[JsonExtensionData]` and perhaps check its count before continuing processing

Reply:
Did that, please take a look. :)